notebooks/500_use_cases/501_dobot/501a_training_a_model_with_cubes_from_a_robotic_arm.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>I have tested this in Windows and this line is not working. The line is working is: <code>%pip install anomalib[full]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/1084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>